### ISSUE TITLE
Having difficulty creating a new species holotype

### ISSUE THREAD

**Issue by mbprondzinski** [mbprondzinski/ALMNH#5]
The new name and classification have been created,  but I get this error message:
![image](https://user-images.githubusercontent.com/43214224/85881535-60372380-b7a3-11ea-9daa-d7c2d03a404e.png)

there are two records referencing this species, ALMNH:ES:1040 and ES:2792

How does this get remedied?  Thanks

**Comment by campmlc** [comment]
Since this a new genus and species just described, it isn't in any of the external taxonomic databases yet, so you'll need to add in the classification.

**Comment by campmlc** [comment]
Looks like you did that - so should be OK.

**Comment by campmlc** [comment]
@dusty I assume the Probably not Valid is the result of not finding this name in any other source?

**Comment by mbprondzinski** [comment]
That's what I was wondering?  Who makes sure it gets to GBIF, etc.?

**Comment by Jegelewicz** [comment]
> Probably not Valid is the result of not finding this name in any other source

Yep - Whoever published the name should notify GBIF I think - let me dig that up...

**Comment by Jegelewicz** [comment]
Go [here](https://data-blog.gbif.org/post/gbif-backbone-taxonomy/) and scroll down to "Missing names and species, what to do about it?" to find out about getting names into the GBIF backbone.

If you have a specimen designated as a type and the classification filled in - the taxon should eventually get added to GBIF on its own.

**Comment by mbprondzinski** [comment]
Thanks, Teresa.  I think I'll wait until AFTER the holiday!  Yes, it is a designated holotype and the classification is filled in.  How long does it usually take before being recognized?

**Comment by Jegelewicz** [comment]
No idea and probably depends on when GBIF is completing a backbone update. In any case - your verbatim identification will appear there, so it should be OK.

**Comment by mbprondzinski** [comment]
Still haven't gotten the newly created taxon affirmed.  Also, the old taxon hierarchy is still showing up as part of the referral specimen ID. (ES:1040)

**Comment by Jegelewicz** [comment]
@mbprondzinski "referral" should be used only for non-peer reviewed citations. Found the PDF! I am changing the "referral" to voucher - the term to be used for things documented in a peer-reviewed paper.

As for the classification (taxon hierarchy) - it was created by Adiel, so if it is not correct, then he should be able to change it. You can send me the correct classification if you want me to change it.

I also added the taxon name Cretasquatina along with its classification.

This is how the record looks to me - is this not the correct classification?

![image](https://user-images.githubusercontent.com/5725767/87344951-83383600-c50c-11ea-901c-9770482e516e.png)

**Comment by mbprondzinski** [comment]
Hi Teresa,

No the holotype is a different specimen (2792), and the hierarchy doesn’t show up there at all because it is “invalid”.  1040 is a referral in the paper. It was previously Pristidae? I think, but I changed that to the new identification, but the old hierarchy is still showing up under the new ID.  And I created the holotype hierarchy, but put Adiel down as the manager (which he is of the Paleo collection).  You should be able to access the paper.  I uploaded it to TACC

**Comment by Jegelewicz** [comment]
@mbprondzinski got it - I know the paper says "referral specimen" but "referral" citations are only used for non-peer reviewed mentions, voucher is used for peer-reviewed ones. (see the definitions at https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcitation_type_status)

As for the classifications, it's crazy, but the one for the holotype shows nothing:

![image](https://user-images.githubusercontent.com/5725767/87345413-40c32900-c50d-11ea-9a32-4a6a706911c7.png)

 while the one for the voucher shows

![image](https://user-images.githubusercontent.com/5725767/87345494-5e908e00-c50d-11ea-9b69-141dba551096.png)

The classification is this:

![image](https://user-images.githubusercontent.com/5725767/87345551-75cf7b80-c50d-11ea-81b8-157cc793368c.png)

which doesn't match either of them!

I cannot assign this to Dusty, because this is your personal repository and he isn't a member - you really should post these things to the Arctos repository.

I'll put a link to this there and see if he can access it.

**Comment by mbprondzinski** [comment]
I was going to put this into the Taxonomy issues, but how the hell do I add an issue to Taxonomy???  I tried but couldn’t figure it out…  GitHub just makes me frustrated.

**Comment by Jegelewicz** [comment]
Go [here](url) and open a blank issue, give it a nice descriptive title and enter the details in the comment box. I'll do the rest!

![image](https://user-images.githubusercontent.com/5725767/87348763-80d8da80-c512-11ea-9847-c815998d954e.png)
